tests/components/advantage_air/test_cover.py

```diff
@@ -49,7 +49,7 @@ async def test_cover_async_setup_entry(hass, aioclient_mock):
     state = hass.states.get(entity_id)
     assert state
     assert state.state == STATE_OPEN
-    assert state.attributes.get("device_class") == DEVICE_CLASS_DAMPER
+    assert state.attributes.get("device_class") == CoverDeviceClass.DAMPER
```

Comment:
I think it might make sense to use `is` rather than `==` now that we are using enums

Reply:
Hmmm... when I do that, pytest fails with:
```
        assert state.state == STATE_OPEN
>       assert state.attributes.get("device_class") is CoverDeviceClass.DAMPER
E       AssertionError: assert 'damper' is <CoverDeviceClass.DAMPER: 'damper'>
E        +  where 'damper' = <built-in method get of mappingproxy object at 0x7f0e5016d3d0>('device_class')
E        +    where <built-in method get of mappingproxy object at 0x7f0e5016d3d0> = mappingproxy({'current_position': 100, 'device_class': 'damper', 'friendly_name': 'Zone open without sensor', 'supported_features': 7}).get
E        +      where mappingproxy({'current_position': 100, 'device_class': 'damper', 'friendly_name': 'Zone open without sensor', 'supported_features': 7}) = <state cover.zone_open_without_sensor=open; current_position=100, device_class=damper, friendly_name=Zone open without sensor, supported_features=7 @ 2021-12-16T09:19:07.680229+00:00>.attributes
E        +  and   <CoverDeviceClass.DAMPER: 'damper'> = CoverDeviceClass.DAMPER

tests/components/advantage_air/test_cover.py:52: AssertionError
```
Investigating...

Reply:
Let's discuss this in https://github.com/home-assistant/core/pull/61989#discussion_r770296057 and mark this as draft until it is finalised